mdot/templates/mdot/developers/request.html

```diff
@@ -18,7 +18,13 @@ <h2 class="mdot-subhead" id="page_header">Request Developer Access</h2>
             {{ form.non_field_errors }}
 
             <h3 class="form-heading">App Information: </h3>
+
             <p class="form-description">This portion of the form should be filled out with the details of the app you would like to submit to the UW App Store.</p>
+
+            {% if managerform.errors or sponsorform.errors or appform.errors %}
+            <p style="color: red">Errors in fields</p>
```

Comment:
We'd like a little clearer messaging here - see http://uxdesign.uw.edu/interaction/forms.html - Something along the lines of "Please correct the field errors below" + bullet list of everything that fails validation.